addons/web_editor/static/src/components/media_dialog/file_selector.xml

```diff
@@ -57,9 +57,10 @@
             validateUrl="validateUrl"
             multiSelect="props.multiSelect"/>
         <t t-call="{{ constructor.attachmentsListTemplate }}"/>
-        <div name="load_more_attachments" class="mt-4 text-center mx-auto o_we_load_more" t-ref="load-more-button">
-            <button t-if="canLoadMore" class="btn btn-primary o_load_more" type="button" t-on-click="handleLoadMore">Load more...</button>
-            <div t-if="hasContent and !canLoadMore" class="mt-4 o_load_done_msg">
+        <div name="load_more_attachments" class="position-fixed pt-3 text-center mx-auto o_we_load_more" t-ref="load-more-button">
+            <div t-if="canScroll" class="d-flex align-items-center mx-auto btn btn-primary rounded-circle fa fa-chevron-down o_scroll_attachments" t-on-click="handleScrollAttachments"/>
+            <button t-if="canLoadMore and !canScroll" class="btn btn-primary o_load_more" type="button" t-on-click="handleLoadMore">Load more...</button>
+            <div t-if="hasContent and !canLoadMore and !canScroll" class="mt-2 o_load_done_msg">
```

Comment:
I feel it easier to read
```xml
            <div t-if="canScroll" class="d-flex align-items-center mx-auto btn btn-primary rounded-circle fa fa-chevron-down o_scroll_attachments" t-on-click="handleScrollAttachments"/>
            <t t-else="">
                <button t-if="canLoadMore" class="btn btn-primary o_load_more" type="button" t-on-click="handleLoadMore">Load more...</button>
                <div t-elif="hasContent" class="mt-2 o_load_done_msg">
                    <span><i t-esc="allLoadedText"/></span>
                </div>
            </t>
```
Simple condition instead of repeating, but again, matter of preferences, if you don't like it you can ignore it.

Reply:
I think the opposite haha
With what I did, you read it like "in french" instead of decomposing the "if, else, if, elif" so it is easier to understand (just my opinion)

Reply:
What's really annoying IMO, is that you have no clue about the fact that those are exclusive, you have no idea that one can't be next to the other, while it's crystal clear with `else` and `elif`.

But yeah, without context and in absolute, only looking at a single condition at a time, it's easier to read.